modules/dxp/apps/portal-workflow/portal-workflow-metrics-service/src/main/java/com/liferay/portal/workflow/metrics/internal/background/task/WorkflowMetricsSLAProcessBackgroundTaskHelper.java

```diff
@@ -107,21 +105,12 @@ private String _getBackgroundTaskName(
 	@Reference
 	private BackgroundTaskLocalService _backgroundTaskLocalService;
 
-	@Reference(target = ModuleServiceLifecycle.PORTLETS_INITIALIZED)
-	private ModuleServiceLifecycle _moduleServiceLifecycle;
```

Comment:
Do not remove ModuleServiceLifecycle.PORTAL_INITIALIZED it's used to ensure the component will be activated after the portal is initialized.

Thanks!